### PR TITLE
NATIVE-448: Translate cancel in overflow menu on ios

### PR DESCRIPTION
**Pull request by klinzo**

### CODE REVIEW

**Comment by maxammann** [comment]
Just tested on iOS, works

E2E checkout is failing for some reason

**Comment by maxammann** [comment]
I restarted the check

File: src/modules/layout/components/MaterialHeaderButtons.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    overflowButtonRef,
+    hiddenButtons,
+    cancelButtonLabel
+  })
+
+const MaterialHeaderButtons = (props: { cancelLabel: string, children: React.Node }) => {
+  const { cancelLabel, ...otherProps } = props
+  return (
+    <HeaderButtons HeaderButtonComponent={MaterialHeaderButton}
+                   OverflowIcon={<MaterialIcon name='more-vert' size={23} color='black' />}
```
Comment (Schedulaar) [inline src/modules/layout/components/MaterialHeaderButtons.js:27]: You could use a color from the theme here if you're motivated.

**Comment by Schedulaar** [comment]
Where do you have the translations from? :thinking:

**Comment by klinzo** [comment]
@Schedulaar from the intro slides section in the locales.

**Pull request merged**